src/nft/components/bag/BagFooter.tsx

```diff
@@ -76,6 +91,16 @@ export const BagFooter = ({
             {`${ethNumberStandardFormatter(totalUsdPrice, true)}`}
           </Row>
         </Column>
+        {showWarning && (
+          <WarningText>
+            <WarningIcon />
+            {!sufficientBalance ? (
+              `Insufficient funds (${formatWeiToDecimal(balance.toString())} ETH)`
```

Comment:
I think what is left is just the warning `Insufficient funds` is all that is needed and not the balance

Reply:
Thanks, double checking this w/ design.